ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -822,18 +822,6 @@ Measure.prototype.defaultOptions = merge(
              * mouse. Can be one of x, y or xy.
              */
             selectType: 'xy',
-            /**
-             * This number defines which xAxis the point is connected to.
-             * It refers to either the axis id or the index of the axis
-             * in the xAxis array.
-             */
-            xAxis: 0,
-            /**
-             * This number defines which yAxis the point is connected to.
-             * It refers to either the axis id or the index of the axis
-             * in the yAxis array.
-             */
-            yAxis: 0,
```

Comment:
`measure.typeOptions.xAxis` is supported: https://api.highcharts.com/highstock/annotations.measure.typeOptions.xAxis

Reply:
Right, totally forgot that annotations still can be drawn in the chart's config...